design/httpproxy-conflict.md

```diff
@@ -0,0 +1,118 @@
+# HTTPProxy Conflict Resolution Design
+
+Status: Draft
```

Comment:
Just make this `Status: Approved` for now. We won't come back and update this later.